components/image/image.css

```diff
@@ -0,0 +1,29 @@
+@import url("../../style-guide/compiled-style-guide.css");
+body {
+  background-color: var(--bg-white);
+  font-family: "Exo", sans-serif;
+}
+
+.txt-highlight {
+  color: var(--primary-color);
+}
+
+.component-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 2rem;
+  background-color: var(--primary-bg);
+}
+.img-responsive{
+    max-width: 100%;
+    height: auto;
+    overflow: hidden;
```

Comment:
yes you can remove it, 
by giving max-width: 100% we are already saying the image to not cross the container with respect to width and we gave height:auto which means take as much height as needed and this may overflow the parent container so I tried in a code sandbox https://mx29w.csb.app/ even after giving overflow:hidden, the image is not hiding, so I saw what overflow actually does https://www.w3schools.com/cssref/pr_pos_overflow.asp, it only hides the content which is inside the element (in our case , inside the image element) so it's of no use here.